src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/NotificationEmailService.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.cmc.claimstore.services.notifications;
+
+import org.quartz.JobDetail;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Recover;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+
+@Service
+public class NotificationEmailService {
+
+    private static final Logger logger = LoggerFactory.getLogger(NotificationEmailService.class);
+    private final ResponseNeededNotificationService responseNeededNotificationService;
+
+    public NotificationEmailService(ResponseNeededNotificationService responseNeededNotificationService) {
+        this.responseNeededNotificationService = responseNeededNotificationService;
+    }
+
+    @Retryable(
+        value = {JobExecutionException.class},
+        backoff = @Backoff(delay = 10000))
+    public void process(JobExecutionContext context) throws JobExecutionException {
```

Comment:
How many times is it retried? and if it fails after all those attempts what happens?
I can't see anything rescheduling the job